sci-mathematics/romannumbers/romannumbers-1.0.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="A long range bidirectional roman to decimal numbers converter"
+DESCRIPTION="This is a bidirectional roman to decimal numbers converter.   \
+The program can convert integers from 1 to 2147483647 with the parentheses\
+notation. It has a simple GUI including a challenging exam mode that features\
+two difficulty levels.                                                                                                      \
```

Comment:
Remove the spaces at the end of "levels."
```diff
-two difficulty levels.                                                                                                      \
+two difficulty levels.
```
PS not using the trailing slash \ at the end also starts a new line :)